src/youbora.js

```diff
@@ -53,6 +53,9 @@ class Youbora extends BasePlugin {
     if (this._youbora.getAdapter()) {
       this._youbora.getAdapter().config = update;
     }
+    if (update.options.customAdsAdapter) {
```

Comment:
you have a mix here - you set in the constructor the config option to get customAdsAdapter on it, but here you try to access it via update.options, remove the options, it should be update.customAdsAdapter